canvas_modules/harness/cypress/integration/canvas/clipboard.js

```diff
@@ -21,6 +21,142 @@ describe("Test clipboard with no link selection enabled", function() {
 		cy.openCanvasDefinition("commentColorCanvas.json");
 	});
 
+	it("Test cut & paste in the right { x,y } positions when using keboard", function() {
+		// cmd+x cut the node into the clipboard
+		cy.clickNode("DRUG1n");
+		cy.shortcutKeysCut();
+
+		// move the mouse into {x,y} & cmd+v paste the node where the cursor is
```

Comment:
I prefer comments to be in correct English with a capital letter at the beginning of the comment. Starting a sentence with a lower case letter makes me wonder whether part of the comment has been accidentally truncated.